docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -167,17 +167,23 @@ The operation was successful if the returned status says `true`.
   Kyma Dashboard
   </summary>
 
-1. Using the left navigation, go back to **Workloads** > **Functions**.
-2. Select your `lastorder` Function and navigate to the **Configuration** tab.
-3. Click on **Add Event Subscription+**.
-4. Provide the following parameters:
-    - **Application name**: `myapp`
+1. Using the left navigation go to **Apllications**.
+2. Click on **Create Application+**.
+3. Provide a name for your application, for example `myapp`.
+4. Go to **Namespaces** and click on `default`.
+5. Using the left navigation, go back to **Workloads** > **Functions**.
+6. Select your `lastorder` Function and navigate to the **Configuration** tab.
+7. Click on **Create Subscription+**.
+8. Provide the following parameters:
+    - **Subscription name**, for example `received-orders`
+    - **Application name**: choose `myapp`
     - **Event name**: `order.received`
     - **Event version**: `v1`
+
     **Event type** is generated automatically; in this example: `myapp.order.received.v1`. 
```

Comment:
```suggestion
    - **Event version**: `v1`

    - **Event type** is generated automatically. For this example, it's `sap.kyma.custom.myapp.order.received.v1`. 
```

1) The **Event type** has changed, too: 
![image](https://user-images.githubusercontent.com/53601578/148782655-b0770c7d-1ccc-4e0a-865a-fba94ab36cf4.png)

2) This is currently rendered incorrectly on the [Website](https://deploy-preview-13004--kyma-project-docs-preview.netlify.app/docs/kyma/preview/02-get-started/04-trigger-workload-with-event/), that's why I added it as another list element. It fits, as it refers to one of the required values. 
![image](https://user-images.githubusercontent.com/53601578/148782178-2aec4699-024e-4873-97f8-ae423402d139.png)